tools/pythonpkg/src/include/duckdb_python/pyconnection.hpp

```diff
@@ -75,7 +75,7 @@ struct DuckDBPyConnection {
 
 	unique_ptr<DuckDBPyRelation> FromCsvAuto(const string &filename);
 
-	unique_ptr<DuckDBPyRelation> FromParquet(const string &filename);
+	unique_ptr<DuckDBPyRelation> FromParquet(const string &filename, bool binary_as_string);
```

Comment:
`ParquetOptions` as well.

Reply:
That stuff comes from python land, I don't think it would make much sense.

Reply:
Ah yeah, fair enough